src/_posts/aframe-v0.3.0.md

```diff
@@ -9,9 +9,9 @@ image:
 
 [webvr1.0]: https://w3c.github.io/webvr/
 
-A-Frame **v0.3.0** has dropped! With support for the **[new WebVR 1.0
+A-Frame **version 0.3.0** has dropped! With support for the **[new WebVR 1.0
 API][webvr1.0]**, we aim higher towards state-of-the-art VR. With room-scale
-tracking, we can walk in a web page, and with tracked controller support, we
+tracking, we can walk in a web page; and with tracked controller support, we
```

Comment:
semicolons are used for emphasizing the previous clause, no? The tracked controller support is a new clause so seems a comma suffices